docs/HowTo/Get-Started/Starting-node.md

```diff
@@ -156,3 +160,38 @@ To run a node on MainNet with the HTTP JSON-RPC service enabled and available fo
 ```bash
 besu --rpc-http-enabled
 ```
+
+## Besu launcher
+
+Use the Besu launcher to interactively configure and start a node with the most common options. The
+launcher asks a series of questions and generates a [configuration file](../Configure/Using-Configuration-File.md).
+
+To run the Besu launcher:
+
+```bash
+besu --Xlauncher
+```
+
+Answer each of question, or press **Enter** to accept the default value.
```

Comment:
```suggestion
Answer each of question, or press ++Enter++ to accept the default value.
```
Will display the key, see https://facelessuser.github.io/pymdown-extensions/extensions/keys/#action-keys